extension-requests/local-utils.js

```diff
@@ -103,8 +103,10 @@ const parseExtensionRequestParams = (uri, nextPageParamsObject) => {
   return nextPageParamsObject;
 };
 
-async function updateExtensionRequest({ id, body }) {
-  const url = `${API_BASE_URL}/extension-requests/${id}`;
+async function updateExtensionRequest({ id, body, dev }) {
+  const url = dev
+    ? `${API_BASE_URL}/extension-requests/${id}?dev=true`
+    : `${API_BASE_URL}/extension-requests/${id}`;
```

Comment:
@pankajjs  I had done it like this (const url =  `${API_BASE_URL}extension-requests/${id}?dev=${dev}`;)  previously but some old tests were failing because of that.

So, should I still change it?

Reply:
- The related test with extension requests are failing?
 - How many of them are failing and why?

Reply:
1. yes 2 tests related to updating extension requests are failing.
2. because we have not intercepted a response for ?dev=true URL

Reply:
Where are these tests?Can you help me navigate?

Reply:
![image](https://github.com/user-attachments/assets/8ec6b892-5146-4428-899e-f506c2fd0616)

these were the 2 tests that were failing.